onadata/settings/travis_test.py

```diff
@@ -67,4 +67,5 @@
 VERIFIED_KEY_TEXT = 'ALREADY_ACTIVATED'
 
 ODK_TOKEN_FERNET_KEY = 'ROsB4T8s1rCJskAdgpTQEKfH2x2K_EX_YBi3UFyoYng='
+ODK_SUBMISSION_URL = ''
```

Comment:
This is not necessary, let us remove it.

Reply:
Removed the `ODK_SUBMISSION_URL` setting in latest commits